.github/workflows/deploy_ecs_relayer1_PROD.yml

```diff
@@ -0,0 +1,52 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: BUSL-1.1
+
+name: Deploy ECS relayer 1
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Pipline should start when release/tag created and request manual approve by admin

Reply:
True, changed, thanks!